elytron/src/main/java/org/wildfly/extension/elytron/TokenRealmDefinition.java

```diff
@@ -148,6 +149,18 @@ static class JwtValidatorAttributes {
                 .setMinSize(1)
                 .build();
 
+        static final SimpleAttributeDefinition CONNECTION_TIMEOUT = new SimpleAttributeDefinitionBuilder(ElytronDescriptionConstants.CONNECTION_TIMEOUT, ModelType.INT, true)
+                .setAllowExpression(true)
+                .setValidator(new IntRangeValidator(0))
+                .setRestartAllServices()
+                .build();
+
+        static final SimpleAttributeDefinition READ_TIMEOUT = new SimpleAttributeDefinitionBuilder(ElytronDescriptionConstants.READ_TIMEOUT, ModelType.INT, true)
+                .setAllowExpression(true)
+                .setValidator(new IntRangeValidator(0))
+                .setRestartAllServices()
+                .build();
+
```

Comment:
Are these attributes only meant to be registered when the server supports PREVIEW features?  If so, you'll want to specify this.
e.g.
```java
.setStability(Stability.PREVIEW)
```